api/src/pcapi/scripts/acceslibre/match_acceslibre.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+import pcapi.core.history.api as history_api
+import pcapi.core.history.models as history_models
+from pcapi.core.offerers import api as offerers_api
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def match_acceslibre(venue: offerers_models.Venue) -> None:
+    offerers_api.set_accessibility_provider_id(venue)
+
+    if not venue.accessibilityProvider:
+        logger.info("No match found at acceslibre for Venue %s ", venue.id)
+        return
+
+    history_api.add_action(
+        history_models.ActionType.INFO_MODIFIED,
+        author=None,
+        venue=venue,
+        comment=f"Recherche automatisée du lieu désormais permanent avec Acceslibre, synchronisé avec {venue.accessibilityProvider.externalAccessibilityUrl}",
```

Comment:
Les actions `INFO_MODIFIED` contiennent toujours un champ `"modified_info"` dans les `extraData`.
```suggestion
        comment=f"Recherche automatisée du lieu désormais permanent avec Acceslibre",
        extraData={"modified_info": {
            "accessibilityProvider.externalAccessibilityId": {
                "old_info": None,
                "new_info": venue.accessibilityProvider.externalAccessibilityId
            },
            "accessibilityProvider.externalAccessibilityUrl": {
                "old_info": None,
                "new_info": venue.accessibilityProvider.externalAccessibilityUrl
            },
        }},
```
Je n'ai pas testé ce code, donc à vérifier :slightly_smiling_face: .
Ou l'ancienne valeur à la place de `None` mais a priori on n'appelle ce job que si c'est vide au départ.
Ainsi les données seront toujours structurées de la même manière, et donc affichées de la même manière dans le BO.